Multisig/UI/App/UIKit/CollectiblesViewController/CollectiblesViewController.swift

```diff
@@ -0,0 +1,112 @@
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.registerCell(CollectibleTableViewCell.self)
+        tableView.registerHeaderFooterView(CollectiblesHeaderView.self)
+        tableView.rowHeight = rowHeight
+        tableView.sectionHeaderHeight = headerHeight
+        tableView.sectionFooterHeight = footerHeight
+        tableView.backgroundColor = tableBackgroundColor
+        tableView.separatorStyle = .none
+
+        emptyView.setText("Collectibles will appear here")
+        emptyView.setImage(#imageLiteral(resourceName: "ico-no-collectibles"))
+    }
+
+    override func reloadData() {
+        super.reloadData()
+        currentDataTask?.cancel()
+        do {
+            let address = try Address(from: try Safe.getSelected()!.address!)
+
+            currentDataTask = clientGatewayService.asyncCollectibles(at: address) { [weak self] result in
+                guard let `self` = self else { return }
+                switch result {
+                case .failure(let error):
+                    DispatchQueue.main.async { [weak self] in
+                        guard let `self` = self else { return }
+                        // ignore cancellation error due to cancelling the
+                        // currently running task. Otherwise user will see
+                        // meaningless message.
+                        if (error as NSError).code == URLError.cancelled.rawValue &&
+                            (error as NSError).domain == NSURLErrorDomain {
+                            return
+                        }
+                        self.lastError = error
+                        self.onError()
+                    }
+                case .success(let collectibles):
+                    let sections = CollectibleListSection.create(collectibles)
+
+                    DispatchQueue.main.async { [weak self] in
+                        guard let `self` = self else { return }
+                        self.sections = sections
+                        self.onSuccess()
+                    }
+                }
+            }
+        } catch {
+            lastError = error
+            onError()
+        }
+    }
+
+    // MARK: - Table view data source
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return sections.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return sections[section].collectibles.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(CollectibleTableViewCell.self, for: indexPath)
+        let collectible = sections[indexPath.section].collectibles[indexPath.row]
+        cell.configure(collectible: collectible)
+        return cell
+    }
+
+    // MARK: - Table view delegate
+
```

Comment:
Currently, if I select a cell, it stays selected. That looks wrong. Can we deselect the cell if we tap it?

Reply:
Or even disable selection

Reply:
I will do in the next PR